runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetConfig.java

```diff
@@ -76,12 +76,12 @@ default boolean highlightUntaggedFish()
 	)
 	@Range(
 		min = 1,
-		max = 60
+		max = 100
```

Comment:
Maybe we can increase this some if we are certain it's around 40-60 secs, 120 maybe?

Reply:
After getting another fishing level yesterday I feel like 60 is ideal. Almost never had a fish which was tagged which didn't actually was and seldomly had a fish which was untagged still go in a net. Which is better in my opinion than a fish who is tagged but actually isn't. At 100 this happens pretty often so I wouldn't go higher than this.